src/shared/components/MLPlayer/MLPlayer.hooks.ts

```diff
@@ -0,0 +1,120 @@
+import { SyntheticEvent, useEffect, useMemo, useRef, useState } from 'react';
+import { OnProgressProps } from 'react-player/base';
+import ReactPlayer from 'react-player';
+
+import { ResourceDataType } from 'shared/components/MediaUploader/MediaUploader.types';
+
+import { MLPlayerStateProps } from './MLPlayer.types';
+
+const calculateTime = (secs: number) => {
+  const minutes = Math.floor(secs / 60);
+  const seconds = Math.floor(secs % 60);
+  const returnedSeconds = seconds < 10 ? `0${seconds}` : `${seconds}`;
```

Comment:
seconds < 10 ? `0${seconds}` : seconds;

Reply:
for ex.: 
9 sec => 0:09
10 sec => 0:10
69 sec => 1:09
70 sec => 1:10